app/views/editorial/submissions/_content.haml

```diff
@@ -34,10 +34,4 @@
         %div.content-block
           != markdown_content content[:content_body]
 
-    - content.optional_keys_in_order.each do |key|
-      %dt= key.to_s.titleize
-      %dd
-        - if content[key].nil?
-          none
-        - else
-          = content[key]
+    = render partial: "editorial/submissions/node_types/#{node_type_template}", locals: { content: content }
```

Comment:
Maybe it would be cleaner if rather than requiring us to make an (empty) file for each node type, (i.e. 3/5 of them are empty at the moment) we only call render partial if `node_type_template` exists?